_posts/2020-04-07-charity-shop-exchange.md

```diff
@@ -0,0 +1,63 @@
+
+**Can you describe your project?**
+
+Charity Shop Exchange offers a subscription service of books & DVDs. People need new things to read and watch - especially when feeling isolated - and our subscription service facilitates this in an economical way. Subscribers receive a (weekly, bi-weekly or monthly)  selection of books and DVDs which have been sourced from local charity shops.
+
+**How did your team come together?**
+
+All of us met through *Slack*, or *Twitter* while trying to discover a way we could help out during this crisis. Kelvin had taken the steps to reach out to us individually, add us to a Slack channel so we could discuss the concept, and from there we got right down to it!
+
+**Can you tell us about your team?**
+
+*Kelvin Lockwood* hails from Bournemouth, UK  
+*Blake Prouty (that's me!)*, am from Sioux Falls, South Dakota  
+*Matt Doyle* lives in Sydney Australia  
+*Dicky Jiang* is based in Bandung Area, West Java, Indonesia
+
+**What skills do they bring?**
+
+*Kelvin Lockwood* - Product Ownership and Product Developer  
+*Blake Prouty (me again!)* - Full Stack Web Developer, HTML, CSS, JS, and React/Node  
+*Matt Doyle* - Full Stack Web Development: HTML, CSS, JS, and React/PHP/WP  
+*Dicky Jiang* - UIX Designer: Photoshop, Photography, Graphic Design, CSS, HTML, and Web Design
+
+**How was the experience of working remotely?**
+
+Could not have been better. Through the use of Github Projects, Google Documents, and Milanote we were able to work through open issues and tasks. Additionally, being that we were all in different time zones, someone was waking up as soon as the other was going to sleep, so we were nearly making progress around the clock.
+
+**How does it feel to be collaborating on a project at a time like this?**
+
+A great benefit, we’ve all shared with each other the additional steps we’ve taken to remain social and cope with the challenges we’ve faced. Whether it’s on a zoom-call or just in Slack, the frequent conversation and working towards a goal is therapeutic through and through. 
+
+**What tools, made available by the  hackathon, are you using?**
+
+At this time we’ve not had a need for a backend component on our project, but would like to thank [FaunaDB](https://fauna.com/) for offering their database services to the hackathon, I’ll be sure to use them in an upcoming project!
+
+**Is there anything else you’d like to say?**
+
+We would just like to thank everyone involved in this collaborative event and for allowing us to share what we’ve built!
+
+Blake *A.* Prouty
+
+For more information on Charity Shop Exchange:
+
+**Website:** [Charity Shop Exchange](https://charityshopexchange.com)  
+**Source Code:** [CSE code](https://github.com/Charity-Shop-Exchange/Charity-Shop-Exchange)  
+**Country:** United Kingdom, USA, Australia and Indonesia
```

Comment:
I think it would be good to end all our blog posts with a blurb to Join the fight as well.  This is the one from the first blog post:

```
If you haven't already signed up, you can [join the fight](https://codevid19.com/#join-the-fight) and more directly from our [website](https://codevid19.com)!
```

Reply:
I could add this to the blog layout template. Alternatively, perhaps this should be in the footer on all pages?

Reply:
Either works for me.  There may be reason to change that parting message too, anything works really from manual to template-ishy!

Reply:
Let's do this in a seperate PR and get this out as is.

I'd like to get the rest of the teams thoughts on adding join the fight to the bottom of all blog posts vs all pages.

It's a good idea!

Are we good with shipping this and starting a seperate PR for this change @ajyong?